docs/manifest/runtime.md

```diff
@@ -40,7 +40,7 @@ For more information, see [Version overrides in the manifest](/office/dev/add-in
 
 |  Element |  Required  |  Description  |
 |:-----|:-----|:-----|
-| [Override](override.md) | No | **Outlook**: Specifies the URL location of the JavaScript file that Outlook Desktop requires for [LaunchEvent extension point](extensionpoint.md#launchevent) handlers. **Important**: At present, you can only define one `<Override>` element and it must be of type `javascript`.|
+| [Override](override.md) | No | **Outlook**: Specifies the URL location of the JavaScript file that Outlook Desktop requires for [LaunchEvent extension point](extensionpoint.md#launchevent) handlers. **Important**: At present, you can only define one **Override** element and it must be of type `javascript`.|
```

Comment:
On [this Style Guide page](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28975), I see:
<img width="796" alt="image" src="https://user-images.githubusercontent.com/7977793/176540688-ec69d213-8ec9-4afa-bd7b-efe3b559b005.png">
It makes me think that we should retain the angle brackets and add bold around them: 

```suggestion
| [Override](override.md) | No | **Outlook**: Specifies the URL location of the JavaScript file that Outlook Desktop requires for [LaunchEvent extension point](extensionpoint.md#launchevent) handlers. **Important**: At present, you can only define one **<Override>** element and it must be of type `javascript`.|
```

What do you think?

Reply:
Sigh. OK. I'll redo both PRs.

Reply:
I'm sorry :( 
Thanks for doing this! It's so tedious.